docs/configuration/index.rst

```diff
@@ -0,0 +1,75 @@
+Configuration
+=============
+
+This is the list of possible configurations that go in ``conf.py``.
+
+Yosys
++++++
+
+``hdl_diagram_yosys`` tells the program what version or binary of Yosys to use.
+By default it is set to ``YoWASP``. Setting it to ``system``, the program will
+use Yosys available in the current PATH. It can also contain the path to a specific
+Yosys binary.::
+
+    hdl_diagram_yosys = "yowasp" # default
+
+    hdl_diagram_yosys = "system" # use yosys from PATH
+
+    hdl_diagram_yosys = "<path-to-Yosys>" # use specific yosys binary
+
+
+netlistsvg
+++++++++++
+
+netlistsvg can take various skin files for use when creating diagrams. It is
+set to ``default`` by default, using the built-in netlistsvg skin.::
+
+    hdl_diagram_skin = "<path-to-skin>"
+
+
+Output format
++++++++++++++
+
+The output format for the generated diagrams can either be set to ``svg`` or ``png``.::
+
+    hdl_diagram_output_format = "svg"
+
+    hdl_diagram_output_format = "png"
+
+
+GHDL
+++++
+
+GHDL can either be prebuilt into Yosys or loaded at runtime. If it is loaded at
+runtime, then this can either be set to ``module`` if GHDL is in the PATH, or
```

Comment:
This is not exactly correct. GHDL must be in the PATH (or it must be otherhow found by the plugin) because it relies on libghdl, which relies on the VHDL libraries (std, IEEE...) in a GHDL installation. However, what needs to be passed to Yosys is the location of the module/plugin, which does also need to be in the PATH if `ghdl` is used only (instead of the full path).